js/src/dropdown.js

```diff
@@ -1,3 +1,5 @@
+/* global Popper */
```

Comment:
I need some context because I'm not confident with the Bootstrap v4 code base, what do you use to bundle the package and convert `import`s?

[Webpack](https://webpack.github.io/docs/library-and-externals.html), [Rollup](https://github.com/rollup/rollup/wiki/JavaScript-API#external) and [Browserify](https://benclinkinbeard.com/posts/external-bundles-for-faster-browserify-builds/) allow to define a dependency as external, thus avoiding to bundle it together with your own source code.